### ISSUE TITLE
SEO

### ISSUE THREAD

**Issue by GreysonElkins** [DougieDev/refactor-fitlit-B#40]
- [ ] not mobile friendly / responsive
- [ ] document does not have meta description
- [ ] img alt tags
